### DISCUSSION TITLE
Discussions are open!

### DISCUSSION THREAD

**Discussion by tagyoureit** [tagyoureit/nodejs-poolController#403] [Announcements]
Hi all,

With the amount of traffic that we are getting, Gitter is getting harder and harder to use.  It isn't easy to find content, add images in threads, link/convert to issues, etc.  To that end, we have decided to open up the Github Discussions.  This should prove to be a much more integrated approach to code and issue management.  

Issue templates are also being updated to better reflect the needs of the community.  

Regards,
@tagyoureit and @rstrouse

**Comment by DrJeff** [comment]
Threaded comments on my iPhone. Awesomeness!!

**Comment by rock-crusher** [comment]
This is a great Idea!
It will be much easier to use especially for a novice like me. 👍
Thanks for all your help in the past & here in the near future!
